wiki/Glossary/en.md

```diff
@@ -798,27 +803,41 @@ Star priority has two meanings:
    - The higher the star priority, the faster it gets noticed.
    - Users that have supporter status active when voting will shoot a pink star (![Kudosu](/wiki/shared/Kudos.png)) that increases Star Priority by 2 each, whilst a user without supporter will shoot a blue star (![Star](/wiki/shared/icon/star.gif)) that is worth only 1 each. Users may be able to shoot multiple stars depending on how many votes they have left.
 
+### Star Rating
```

Comment:
please update the redirect `difficulty_stars` to point it towards `Glossary#star-rating`